Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -20,7 +20,7 @@ def create_graph(self, nr_of_nodes, nr_of_edges):
         :param nr_of_nodes: amount of nodes in the random graph
         :param nr_of_edges: amount of edges in the random graph
         """
-        self.graph = gnm_random_graph(nr_of_nodes, nr_of_edges)
+        self.graph = gnm_random_graph(nr_of_nodes, nr_of_edges, directed=True)
```

Comment:
Please build the same, original graph without this method to keep the test the same.